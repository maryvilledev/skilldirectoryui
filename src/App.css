.App {
  text-align: left;
}

.App-header {
  background-color: #666;
  height: 30px;
  padding: 20px;
  padding-left: 10px;
  padding-top: 5px;
  color: white;
}

.App-intro {
  font-size: large;
}

h1 {
  color: rgb(40, 185, 217);
  font-size: 23px;
  margin:0px;
  padding: 0px;
}
h3 {
  font-size: 12px;
  margin:0px;
  padding:0px;
}
