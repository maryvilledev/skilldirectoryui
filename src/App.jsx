import React from 'react';
import {
  browserHistory,
  IndexRedirect,
  Route,
  Router
} from 'react-router';

import Auth from './Auth';
import Home from './Home';
import Skills from './Skills';
import Header from './Header';
import Team from './Team';

function App() {
  return (
    <Router history={browserHistory}>
      <Route path="/" component={Header}>
        <IndexRedirect to="/home" />
        <Route path="home" component={Home} />
        <Route path="skills(/:id)" component={Skills} />
        <Route path="team(/:id)" component={Team} />
        <Route path="auth" component={Auth} />
      </Route>
    </Router>
  );
}

export default App;
