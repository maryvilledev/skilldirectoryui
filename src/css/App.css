.App {
  text-align: left;
}

.App-header {
  background-color: #555555;
  height: 80px;
  padding: 20px;
  padding-left: 10px;
  padding-top: 5px;
  color: white;
}

.App-intro {
  font-size: large;
}

h1 {
  color: rgb(40, 185, 217);
  font-size: 23px;
  margin:0px;
  padding: 0px;
  width: inherit;
}
h3 {
  font-size: 12px;
  margin:0px;
  padding:1px;
}

body { padding-top: 70px; }
