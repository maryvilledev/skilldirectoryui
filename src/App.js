import React, { Component } from 'react';
import ReactDOM from 'react-dom';

import { hashHistory, IndexRedirect, Route, Router } from 'react-router';
import 'bootstrap/dist/css/bootstrap.css';

import Skills from './Skills';
import Header from './Header'
import Team from './Team'

class Home extends Component {
  render() {
    return (
      <h1>Skill Directory Home</h1>
    );
  }
}

function App(props) {
  return (
    <Router history={hashHistory}>
      <Route path="/" component={Header}>
        <IndexRedirect to="/home" />
        <Route path="home" component={Home} />
        <Route path="skills" component={Skills} />
        <Route path="team" component={Team} />
      </Route>
    </Router>
  );
}

export default App