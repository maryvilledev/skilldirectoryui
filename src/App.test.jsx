import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import App from './App';

describe('<App />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App />, div);
  });
 
  it('matches the stored snapshot', () => {
    const wrapper = shallow(<App />);
    expect(wrapper.get(0)).toMatchSnapshot();
  });
});
