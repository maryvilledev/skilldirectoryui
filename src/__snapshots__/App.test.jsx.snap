exports[`<App /> matches the stored snapshot 1`] = `
<Router
  history={
    Object {
      "createHref": [Function],
      "createKey": [Function],
      "createLocation": [Function],
      "createPath": [Function],
      "getCurrentLocation": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "listen": [Function],
      "listenBefore": [Function],
      "push": [Function],
      "replace": [Function],
      "transitionTo": [Function],
    }
  }
  render={[Function]}>
  <Route
    component={[Function]}
    path="/">
    <IndexRedirect
      to="/home" />
    <Route
      component={[Function]}
      path="home" />
    <Route
      component={[Function]}
      path="skills(/:id)" />
    <Route
      component={[Function]}
      path="team(/:id)" />
    <Route
      component={[Function]}
      path="auth" />
  </Route>
</Router>
`;
